instance/tests/models/test_openedx_theme_mixins.py

```diff
@@ -65,6 +66,7 @@ def make_test_application(instance, user):
         )
         return application
 
+    @override_settings(INSTANCE_STORAGE_TYPE='s3')
```

Comment:
@pkulkark, wouldn't it be easier and cleaner to just change the default in `.env.test` instead of changing it in multiple places?

Reply:
@lgp171188 I suppose so. I was following the [suggestion](https://tasks.opencraft.com/browse/SE-2245) made in the ticket :). I'll update it.

Reply:
@lgp171188 changing `.env.test` resulted in more test failures so I went back to changing it only for the ones that needed it.